compose/config/types.py

```diff
@@ -92,6 +92,8 @@ def parse_restart_spec(restart_config):
 
 
 def serialize_restart_spec(restart_spec):
+    if not restart_spec:
+        return ''
```

Comment:
Hmm, I was thinking this would be `None`, I'm not sure if this works just as well.